Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadSelectionBinding.cs

```diff
@@ -51,24 +45,21 @@ private void TryRegisterDocumentForSelection(Document? document)
       return;
     }
 
-    if (!_visitedDocuments.Contains(document.Name))
+    if (!_visitedDocuments.Contains(document))
```

Comment:
Why was this change from `name` -> `document`?

Are we sure this leads to the same behaivour?

Reply:
And we don't appear to have a `OnDocumentDestroyed` listener anymore, so this logic is definitely different.

Reply:
name is not safe to use, because everytime you opened a new blank file, they named as `Untitled.dwg` etc